eventlistener.go

```diff
@@ -48,6 +42,31 @@ func (c *Camera) EventListener(done chan bool, room *Room) {
 
 				fmt.Printf("%+v", remoteSdp)
 
+				mediaEngine := webrtc.MediaEngine{}
+				if err := mediaEngine.PopulateFromSDP(remoteSdp); err != nil {
+					fmt.Println("webrtc could not create media engine.", err)
+					continue
+				}
+
+				var h264PayloadType uint8
+				for _, videoCodec := range mediaEngine.GetCodecsByKind(webrtc.RTPCodecTypeVideo) {
+					if videoCodec.Name == "H264" {
```

Comment:
`videoCodec.Name` might include the profile of H264 as well, so I would try calling printf on `videoCodec.Name` and see what you can get.